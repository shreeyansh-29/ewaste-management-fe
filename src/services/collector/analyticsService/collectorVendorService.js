import {COLLECTOR_ANALYTICS_V4} from "../../../container/constant/constant";
import api from "../../../core/utilities/httpProvider";

export const collectorVendorService = () => {
  return api.get(COLLECTOR_ANALYTICS_V4);
};
