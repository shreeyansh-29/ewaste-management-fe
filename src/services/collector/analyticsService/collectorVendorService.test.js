/* eslint-disable no-undef */

import React from "react";
import Enzyme, {shallow} from "enzyme";
import {collectorVendorService} from "./collectorVendorService";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorVendorService", () => {
  const Service = collectorVendorService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
