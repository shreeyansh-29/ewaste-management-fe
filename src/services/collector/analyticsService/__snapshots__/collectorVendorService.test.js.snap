// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorVendorService 1`] = `<Component />`;
